roles/glance/tasks/main.yml

```diff
@@ -43,6 +43,12 @@
     - etype: other
       permission: r
 
+- name: install bbc glance plugins
+  pip:
+    name: "git+git://github.com/blueboxgroup/bbc-openstack-plugins.git#egg=bbc_openstack_plugins"
```

Comment:
We need to use post_python_dependency approach. See, keystone (JellyRoll), Heat(UCD Plugin), Horizon(Customizations). We also will need to get pip module build and uploaded to the mirror. Talk with @zfeldstein